packages/block-library/src/shortcode/index.php

```diff
@@ -21,7 +21,7 @@ function render_block_core_shortcode( $attributes, $content ) {
  * Registers the `core/shortcode` block on server.
  */
 function register_block_core_shortcode() {
-	$path     = __DIR__ . '/shortcode.json';
+	$path     = realpath( __DIR__ . '/block.json' );
```

Comment:
Can you explain why this is needed?

Reply:
I guess it isn't that much necessary. It ensures the canonical absolut pathname.

Reply:
`realpath()` helps resolve symlinks which (as far as I remember) fixes some issues on some servers/OS. Ideally this will use WP's "canonical" path when merged, starting with `ABSPATH`, etc. (ABSPATH can be set for each site/install "by hand" in wp-config.php).